### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by CalvinYing** [kevster555999/DataRocks#10]
The team's goal is to come up with an algorithm that could create an image-to-text converter. In the end, the team did not succeed in finding an algorithm to produce the desired outcome. This project is very ambitious and tough, but the team were still able to apply multiple techniques learned in class, such as regression, k-means clustering, PCA and spectral clustering to construct a model that would help them achieve their goal. The project is very creative and gives students an idea of how to apply machine learning algorithms on images.

Something you may consider:
1. Even when the team predicted texts from images using your training data set, your linear regression model predicted only 88/5055 characters correctly. The team could perhaps try polynomial regression to see if different orders of the predictors would perform better. Also, the team could add loss functions or use other techniques learned in class to increase the accuracy of your model.

2. The team did not explain how they came up with the ideal number of clusters when they performed k-means clustering. Instead of arbitrarily assigning 62 clusters, perhaps the team could use cross validation to find out the optimal number of clusters.
